Include/internal/pycore_interp.h

```diff
@@ -242,7 +242,7 @@ struct _is {
     PyObject *audit_hooks;
     PyType_WatchCallback type_watchers[TYPE_MAX_WATCHERS];
     PyCode_WatchCallback code_watchers[CODE_MAX_WATCHERS];
-    PyContext_WatchCallback context_watchers[CONTEXT_MAX_WATCHERS];
+    PyObject *context_watchers[CONTEXT_MAX_WATCHERS];
```

Comment:
Does this need a `Py_VISIT` somewhere?  I didn't see any for the other fields in this struct, but maybe the GC does something special with this struct?